package.json

```diff
@@ -50,6 +50,7 @@
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
     "@readme/eslint-config": "^4.0.0",
+    "@svgr/webpack": "^5.5.0",
```

Comment:
Any specific reason you like this @svgr/webpack loader over other ones?

Reply:
The integration was pretty simple & had a good number of stars/downloads. It'd be pretty easy to switch it out if there's ever an issue with tit.